### PR TITLE
fix(outlineThickness): allow 0 thickness per segment index

### PR DESCRIPTION
**Pull request by sedghi**
### Context

Incorrect logic prevents each segment from having a 0 outline. The previous logic defaulted to the first segment when the thickness was 0. The accurate approach should be to default to the first segment when it is undefined, as it should be permissible to set any segment's thickness to 0.

### Results

`labelMap.actor.getProperty().setLabelOutlineThickness([3, 0]);` (segment 1 (red)-> 3 , segment 2 (green) -> 0)

**Previously** 

CleanShot 2024-04-23 at 22 48 31@2x

**After**

CleanShot 2024-04-23 at 22 47 55@2x

### Changes

- [ ] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [x] This change adds or fixes unit tests 
- [x] Tested environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**:

### CODE REVIEW

**Comment by sedghi** [comment]
I'm not sure why the test on firefox fails, but for chrome passes

**Comment by sedghi** [comment]
@finetjul sorry for the mention, but are you around to review this quick fix?

File: Sources/Rendering/OpenGL/ImageMapper/index.js [hunk 1]
```diff
@@ -1376,9 +1376,11 @@ function vtkOpenGLImageMapper(publicAPI, model) {
       // Assuming labelOutlineThicknessArray contains the thickness for each segment
       for (let i = 0; i < lWidth; ++i) {
         // Retrieve the thickness value for the current segment index.
-        // If the value is undefined, null, or 0, use the first element's value as a default.
+        // If the value is undefined, use the first element's value as a default, otherwise use the value (event if 0)
```
Comment (finetjul) [inline Sources/Rendering/OpenGL/ImageMapper/index.js:1379]: (event if 0) -> (even if 0)

File: Sources/Rendering/OpenGL/VolumeMapper/index.js [hunk 2]
```diff
@@ -1782,9 +1782,12 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
       // Assuming labelOutlineThicknessArray contains the thickness for each segment
       for (let i = 0; i < lWidth; ++i) {
         // Retrieve the thickness value for the current segment index.
-        // If the value is undefined, null, or 0, use the first element's value as a default.
+        // If the value is undefined, use the first element's value as a default, otherwise use the value (event if 0)
```
Comment (finetjul) [inline Sources/Rendering/OpenGL/VolumeMapper/index.js:1785]: (event if 0) -> (even if 0)

**Review by finetjul — APPROVED** [review]
LGTM, minor typo though

File: Sources/Rendering/OpenGL/ImageMapper/index.js (hunk 1 above)
Comment (sedghi) [reply Sources/Rendering/OpenGL/ImageMapper/index.js]: Thanks updated both

**Comment by sedghi** [comment]
@finetjul Can we merge this? We're blocked from releasing until this is resolved unfortunatelly

**Comment by finetjul** [comment]
> @finetjul Can we merge this? We're blocked from releasing until this is resolved unfortunatelly

Sure thing. (I was on vacation last week).

**Pull request merged**